docs/user/discover.asciidoc

```diff
@@ -285,6 +281,12 @@ To share your findings with a larger audience, click *Share* in the *Discover* t
 
 * <<discover-search-for-relevance, Search for relevance>>.
 
+* <<alerting-getting-started, Create an alert from Discover>>.
+In the toolbar, click *Alerts*, and then
```

Comment:
One important thing we should mention. The alert rule form is pre-populated by data view, query, filters of the last request to ElasticSearch of the Discover main view.

A recommendation would be to 
1. Build a data view, query, filters that matches your need for the data you wanna get an alert notification for, then to click "Create search threshold rule". Of course you could modify the alert rule configuration later on